src/main/java/uk/ac/cam/cl/dtg/segue/api/monitors/SegueLoginbyIPMisuseHandler.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2015 Stephen Cummins
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.cam.cl.dtg.segue.api.monitors;
+
+import com.google.inject.Inject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.ac.cam.cl.dtg.segue.api.Constants;
+import uk.ac.cam.cl.dtg.segue.comm.EmailManager;
+import uk.ac.cam.cl.dtg.util.PropertiesLoader;
+
+import static uk.ac.cam.cl.dtg.segue.api.Constants.NUMBER_SECONDS_IN_ONE_HOUR;
+import static uk.ac.cam.cl.dtg.segue.api.Constants.NUMBER_SECONDS_IN_TEN_MINUTES;
+import static uk.ac.cam.cl.dtg.util.LogUtils.sanitiseLogValue;
+
+/**
+ * Handler to detect bruteforce login attempts.
+ * 
+ * Preventing users from overusing this endpoint is important as they may be trying to brute force someones password.
+ *
+ */
+public class SegueLoginbyIPMisuseHandler implements IMisuseHandler {
+    private static final Logger log = LoggerFactory.getLogger(SegueLoginbyIPMisuseHandler.class);
+
+    public static Integer SOFT_THRESHOLD;
+    public static Integer HARD_THRESHOLD;
+    public static Integer ACCOUNTING_INTERVAL;
+
+    private PropertiesLoader properties;
```

Comment:
Unused variables.

Reply:
Ah, I based it on the pre-existing SegueLoginByEmailMisuseHandler, which also has those properties. They appear to be unused there as well though...
Should I remove them for both or should I add an email to the threshold actions as per the other handlers that have those?

Reply:
IMO don't remove anything existing without good unit test coverage of the class ;)

Reply:
Since it is actually unused beyond initially defining it in this case, I'd say it's probably safe to remove. Whether we want to remove it or to implement something that does use it is another matter. Any recommendations either way?

Reply:
Okay, I've checked with Dave and for the time being we're going to leave the threshold actions as-is. If it gets revisited in future, it will be in a new ticket. As such, I've cleaned up the unused properties.